optimum/habana/transformers/training_args.py

```diff
@@ -437,6 +437,24 @@ def __post_init__(self):
             if version.parse(version.parse(torch.__version__).base_version) < version.parse("2.0.0"):
                 raise ValueError("--optim adamw_torch_fused requires PyTorch 2.0 or higher")
 
+        if (self.torch_compile_mode is not None or self.torch_compile_backend is not None) and not self.torch_compile:
+            assert get_habana_frameworks_version().minor > 12, "Torch compile is not available"
+            self.torch_compile = True
+            assert int(torch.__version__.split(".")[0]) >= 2, "Graph mode is available only in PyTorch 2.x."
```

Comment:
Can we not assume that the version of torch will necessarily be >= 2.x and remove this test? The last Docker image with a version of torch < 2.x was released quite a long time ago I think.